src/test/java/rest/gorestcoIn/APIRest.java

```diff
@@ -0,0 +1,63 @@
+package rest.gorestcoIn;
+
+import io.restassured.RestAssured;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import rest.gorestcoIn.utils.User;
+import rest.gorestcoIn.utils.helpermethods.APICal;
+
+public class APIRest{
+
+    @BeforeClass
+    public void setup() {
+        RestAssured.baseURI = "https://gorest.co.in/";
+        RestAssured.basePath = "public-api/";
+
+    }
+
+    @Test
+    public void postUser() {
```

Comment:
please split the test case and make it responsible for a single functionality check, so you should have ~3 test methods, accordingly responsible for get, post, delete